### PR TITLE
RDS Instance

### PR DESCRIPTION
**Pull request by warrenbailey**
**_What**_
1. Creating an MutliAZ HA database instance
2. Creating a new subnet and renaming the WAF one (so we have two database subnets)
3. Constructing the DATABASE URL and configuring the Survey Runner to use SERVER SIDE STORAGE.

**_How to test**_
1. Check out this branch
2. Modify the deploy_surveyrunner.sh script and point it at 'rds-integration'
3. Deploy a terraform environment
4. Check you can complete the survey
5. Check you can save the survey and come back to it later

**_Who can review**_
Anyone apart from @warrenbailey

### CODE REVIEW

File: vpc.tf [hunk 1]
```diff
@@ -189,15 +190,26 @@ resource "aws_subnet" "sr_application" {
...
+    Name = "${var.env}-database-1-subnet"
+  }
+}
+
+resource "aws_subnet" "database-2" {
+  vpc_id                  = "${aws_vpc.default.id}"
+  cidr_block              = "${var.database_2_cidr}"
+  availability_zone       = "eu-west-1c"
+  tags {
+    Name = "${var.env}-database-1-subnet"
```
Comment (collisdigital) [inline vpc.tf]: database-2?

Comment (warrenbailey) [inline vpc.tf]: It needs two subnets for the multi-AZ functionality to work. The backup goes in the second subnet in a different availability zone. 

I can rename it the variable if it's not clear, maybe database-zone-2?

Comment (warrenbailey) [inline vpc.tf]: Actually just spotted the typo

**Comment by weapdiv-david** [comment]
LGTM :+1:

**Pull request merged**